### ISSUE TITLE
Upper arm pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10793]
**Preferred term label:**
Upper arm pain

**Synonyms**
Pain anywhere between the shoulder and elbow; proximal arm pain.  This is analogous to forearm pain that got https://hpo.jax.org/browse/term/HP:6000299 but that is missing the word pain in the title as previously reported in https://github.com/obophenotype/human-phenotype-ontology/issues/10784

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ulnar shaft fracture
Intersection syndrome of hand tendons
Galeazzi fracture
Radial tunnel syndrome
Wrist dislocation

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001139
- label: Upper arm pain
- parents: [Upper limb pain (HP:0012513)](https://hpo.jax.org/app/browse/term/HP:0012513)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the upper arm.
- comment: Pain anywhere between the shoulder and elbow. Upper arm pain may be observed with conditions such as ulnar shaft fracture, intersection syndrome of hand tendons, Galeazzi fracture, radial tunnel syndrome, and wrist dislocation.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10793
- synonyms: Proximal arm pain(NONE)
